src/main/java/site/coach_coach/coach_coach_server/routine/service/RoutineService.java

```diff
@@ -84,18 +86,20 @@ public RoutineListDto getRoutineList(Long userIdParam, Long coachIdParam, Long u
 			routines = routineRepository.findByUser_UserIdAndCoach_CoachId(routineCreatorDto.userId(),
 				routineCreatorDto.coachId());
 		}
-
+		ZonedDateTime nowInKorea = ZonedDateTime.now(ZoneId.of("Asia/Seoul"));
 		routines.forEach((routine) -> {
```

Comment:
forEach를 사용하여 루프를 돌고, 그 안에서 여러 작업을 처리하는 방식도 잘 동작하지만, stream API를 활용하여 더 간결하고 가독성 좋은 방식으로 코드를 개선할 수 있을 것 같아서요! 
```suggestion
        routines.stream()
            .map(RoutineDto::onvertToDtoWithoutNullAction)
            .filter(dto -> dto.repeats().contains(nowInKorea.getDayOfWeek()))
            .forEach(dto -> {
                if (dto.isCompleted()) {
                    numberOfCompletedRoutine++;
                }
                routineListDto.routineDtos().add(dto);
            });
```